js/app.js

```diff
@@ -0,0 +1,172 @@
+((window,document,undefined)=>{
+  let index=0;
+    const firstAttribute=document.getElementById('firstValue');
+    const secondAttribute=document.getElementById('secondValue');
+    const thirdAttribute=document.getElementById('thirdValue');
+    const fourthAttribute=document.getElementById('fourthValue');
+    
+    const nextBttn = document.getElementById('next');
+    const previousBttn=document.getElementById('previous');
+
+    
+
+     function categories(categoryName,index)
+    {
+
+    const swapiAPI = (index = 1) => `https://swapi.co/api/${categoryName}/${index}/?format=json`;
+    
+    function getPeople(index = 1) {
```

Comment:
rename getPeople to getRecord as it is used for all

Reply:
ok